doc/help/elektra-cmerge-strategy.md

```diff
@@ -0,0 +1,32 @@
+# elektra-cmerge-strategies(7) -- how to merge key sets
```

Comment:
As you know as end result only your merging should exist. It is okay if you duplicate everything for the transition period. But you should at least start writing info texts for people what do use and what our plans are (that only one merge exists and that it will be what is called cmerge but it will be renamed to merge).